frontend/public/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -31,9 +36,21 @@ const withFormState = (Component) => {
       const updateChannel = !_.isEmpty(_.get(nextProps.packageManifest, 'data'))
         ? (nextProps.packageManifest.data.status.channels.find(ch => ch.name === nextProps.packageManifest.data.status.defaultChannel) || nextProps.packageManifest.data.status.channels[0]).name
         : null;
+      const installMode = !_.isEmpty(_.get(nextProps.packageManifest, 'data'))
```

Comment:
Determine if Operator package supports running in `all-namespaces` or just a single namespace.